src/guides/reset_with_sd_card.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: troubleshooting-guide
+description: Home Assistant Green
+guideName: Resetting the device using an SD card
+tools:
+  title: Prerequisites
+  image: green_sd_card.png
+  content: |
+    <ul>
+        <li>Home&nbsp;Assistant Green</li>
+        <li>Home Assistant 13 or older and you cannot upgrade to Home Assistant 13.1 or later</li>
+        <li>Micro SD card</li>
+        <li>PC with internet access</li>
+        <li>You have <a href="/guides/create-backup/"
+          target="_blank">created a backup</a> and stored it on another system.</li>
+    </ul>
+---
+
+<div class="container">
+  <div><h1 class="text-center"></h1></div>
+  <div class="layout horizontal guides">         
+      <div class="layout vertical">
+        <p>Perform this procedure if you want to do a factory reset and you cannot upgrade to Home Assistant 13.1 or later. </p>
```

Comment:
This method is always kinda "deeper" reset since it does reinstall everything from scratch. I would not judge too much on when to use this, if folks feel comfortable handling all these steps with SD card etc, they should feel free to use this method. :man_shrugging: 

```suggestion
        <p>Perform this procedure if you want to do a factory reset your device.</p>
```